cmd/api/src/database/analysisrequest.go

```diff
@@ -86,7 +86,7 @@ func (s *BloodhoundDB) setAnalysisRequest(ctx context.Context, requestType model
 	} else {
 		// Analysis request existed, we only want to overwrite if request is for a deletion request, otherwise ignore additional requests
 		if analReq.RequestType == model.AnalysisRequestAnalysis && requestType == model.AnalysisRequestDeletion {
-			updateSql := `update analysis_request_switch set requested_by = ?, request_type = ?, requested_at = ? limit 1;`
+			updateSql := `update analysis_request_switch set requested_by = ?, request_type = ?, requested_at = ?;`
```

Comment:
👀 Did this sneak in from main or is this intentional here?

Reply:
This is also addressing this ticket as I stumbled upon it. https://specterops.atlassian.net/browse/BED-4950

I'll link the ticket in the description (or I can take this out and make a separate MR if that's more appropriate)